spec/spec_helper.cr

```diff
@@ -6,12 +6,32 @@ Granite::Connections << Granite::Adapter::Mysql.new(name: "mysql", url: ENV["MYS
 Granite::Connections << Granite::Adapter::Pg.new(name: "pg", url: ENV["PG_DATABASE_URL"])
 Granite::Connections << Granite::Adapter::Sqlite.new(name: "sqlite", url: ENV["SQLITE_DATABASE_URL"])
 
-Spec.before_each do
-  Granite.settings.default_timezone = Granite::TIME_ZONE
-end
-
 require "spec"
 require "../src/granite"
 require "../src/adapter/**"
 require "./spec_models"
 require "./mocks/**"
+
+Spec.before_suite do
+  Granite.settings.default_timezone = Granite::TIME_ZONE
+  {% if flag?(:spec_logs) %}
```

Comment:
That's an interesting way to get more detailed logging without having to change much in the specs themselves